roles/jupyterhub/tasks/install.yml

```diff
@@ -18,9 +18,11 @@
   register: df1
 
 
-- name: "Install package: python3-psutil"
+- name: "Install package: python3-packaging"
   package:
-    name: python3-psutil
+    name:
+      - python3-packaging    # 2025-02-15: Debian 13 needs python3-packaging and does NOT need python3-psutil.  Debian 12 doesn't need python3-packaging OR python3-psutil.  Ubuntu 24.04+ comes with python3-packaging pre-installed.
```

Comment:
Instead of apt could "pip install packaging" below with pip and wheel or is it a case of just not having the wheel available?

Reply:
Debian 13 (and possibly Ubuntu 24.04+) are unable to set up the initial venv, if python3-packaging hasn't been installed.

(So there's no opportunity to run `` inside the venv.)

Reply:
Seems like python3.12 teething pains on the pre-release of Debian13

Reply:
Well Ubuntu plucky 25.04 devel install python3.13 and python3-venv installs without pulling in python3-packaging which was not preinstalled, was not preinstalled on 24.04 and python3-venv doesn't pull it in either.

What version of python3 is Debian-13 currently installing?

Reply:
> Debian 13 (and possibly Ubuntu 24.04+) are unable to set up the initial venv, if python3-packaging hasn't been installed.
> 
> (So there's no opportunity to run `` inside the venv.)

So how does ansible get installed then?

Reply:
> What version of python3 is Debian-13 currently installing?

Ubuntu 25.04 and Debian 13 are both currently using Python 3.13.2

Hypothetically it's another case of having to pip install incrementally, in batches.  I'll look later today.

Reply:
Don't feel like searching for what you used in the past, mind sharing you multipass invocation to install Debian-13?

Reply:
https://github.com/iiab/iiab/issues/3399#issue-1407189520

Reply:
Yes.  I've been using:

`multipass launch https://cloud.debian.org/images/cloud/trixie/daily/latest/debian-13-generic-amd64-daily.qcow2 -n deb13-jup -c 2 -m 2G -d 20G`